packages/ember-glimmer/tests/integration/custom-component-manager-test.js

```diff
@@ -0,0 +1,27 @@
+import { moduleFor, RenderingTest } from '../utils/test-case';
+import TestComponentManager from './helpers/test-component-manager';
```

Comment:
We should not attempt to extend from the curly manager, but create a new one that satisifies [this interface](https://github.com/glimmerjs/glimmer-vm/blob/9fb4cd315589186ce7179bed4015d56764001ee7/packages/%40glimmer/runtime/lib/component/interfaces.ts#L21-L86).